model_zoo/DROID-MVP/readme.md

```diff
@@ -0,0 +1,36 @@
+# DROID-MVP Inference Example
+
+This is a simple example script demonstrated how to load and run the DROID-MVP model. Model training and inference was performed using the code provided in the ML4H [model zoo](https://github.com/broadinstitute/ml4h/tree/master/model_zoo/DROID). The example below was adapted from the DROID inference code.
```

Comment:
"This is a simple example script demonstrated" -> "This simple example script demonstrates"